### PR TITLE
Created Job project

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

**Comment by jSylvestre** [comment]
@srkirkland Take a look. The SQL worked when I tested in a controller.

File: EmailNotificationDaily/Functions.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+
+namespace EmailNotificationDaily
+{
+    public class Functions
+    {
+        // This function will get triggered/executed when a new message is written 
+        // on an Azure Queue called queue.
+        public static void ProcessQueueMessage([QueueTrigger("queue")] string message, TextWriter log)
```
Comment (srkirkland) [inline EmailNotificationDaily/Functions.cs]: @jSylvestre there are 3 identical files like this, what's the point of any of them?

File: Commencement.Jobs.NotificationsCommon/ProcessNotifications.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                {
+                    var emailTransmission = new Transmission
+                    {
+                        Content = new Content
+                        {
+                            From =
+                                new Address
+                                {
+                                    Email = "commencement-notify.ucdavis.edu",
+                                    Name = "commencement-notify"
```
Comment (srkirkland) [inline Commencement.Jobs.NotificationsCommon/ProcessNotifications.cs]: @jSylvestre Maybe make this friendlier? "UCD Commencement Notification" or whatever

File: EmailNotificationDaily/Functions.cs (hunk 1 above)
Comment (jSylvestre) [inline EmailNotificationDaily/Functions.cs]: @srkirkland I think they go added automatically when I created the jobs. I can delete them.

**Pull request merged**